api/src/main/java/ai/djl/repository/zoo/Criteria.java

```diff
@@ -181,6 +181,9 @@ public ZooModel<I, O> loadModel()
                 }
             }
         }
+        logger.debug(
```

Comment:
API is an abstract layer which doesn't have any knowledge of underlying engine, platform and model zoo.

This is not the right place to print out this message.

I understand we need to improve model loading message to help troubleshoot model url and model loading related issue. Currently we have to turn on debug log to understand what happened during model loading, this is not a good user experience. We aware of this limitations and looking for an improvement.

I created a PR to address partial of this issue: https://github.com/deepjavalibrary/djl/pull/2678

Reply:
Pull request looks good. Thankyou.